Makefile

```diff
@@ -25,15 +25,15 @@ $(TEMP_TEST_ANALYZER_DIR)/*:
 GO_TEST_FLAGS?=
 
 .PHONY: test
-test:
+test: gofail-enable
```

Comment:
Please don't add `gofail-enable` as a side effect of commands.
Just have integration test workflow enable it.

Reply:
It includes `integration` test workflow in the `make test` which is `PASSES="unit integration release e2e" ./scripts/test.sh $(GO_TEST_FLAGS)` :\ 

I tried make test but it failed with gofail is not enabled error message. How about I add to functional_test_pass? 

https://github.com/etcd-io/etcd/blob/ac61b36e069dd3daeca5d77257931a207b8d4545/scripts/test.sh#L122-L126

Reply:
Second thought, the specific test is skipped now. We can revisit this once it is enabled later.

Reply:
Like I said, do it in github workflow.